### PR TITLE
build and golden masters for tests

### PR DESCRIPTION
**Pull request by NolaDonato**
1. Tests use common.gradle
2. Added delays to make render tests more reliable
3. Fixed path to golden masters

SXR DCO signed off by: Nola Donato nola.donato@samsung.com

### CODE REVIEW

**Comment by thomasflynn** [comment]
cherry-picked this pull-request.  cannot build.  get:
* Where:
Build file '/home/flynnt/trees/sxr/sxrsdk-tests/asset-tests/app/build.gradle' line: 45

* What went wrong:
A problem occurred evaluating project ':asset-tests'.
> Could not find method implementationDebug() for arguments [{name=x3d-debug, ext=aar}] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.

**Comment by NolaDonato** [comment]
I could not reproduce your problem but nevertheless tried to fix it by looking at other build.gradle files. My shot in the dark has been committed, please try to see if it builds for you now.

**Comment by liaxim** [comment]
@thomasflynn Currently the tests build if the tests and the x3d and the script extensions are part of the framework workspace. Will look into fixing the standalone build.

**Comment by liaxim** [comment]
Depends on https://github.com/sxrsdk/sxrsdk/pull/61

**Comment by liaxim** [comment]
The tests are in decent shape. TestSendEvents crashes for me.

File: framework-tests/app/src/androidTest/java/com/samsungxr/stress/StressTests.java [hunk 1]
```diff
@@ -140,18 +141,20 @@ public void gcOomTest2() throws Exception {
...
-            SXRContext sxrContext = ActivityRule.getActivity() .getSXRContext();
+            SXRContext sxrContext = ActivityRule.getActivity().getSXRContext();
+            SXRTexture texture = new SXRTexture(sxrContext);
+
             for (int count = 0; count < MaxInstances; count++) {
                 Log.d(TAG, "Count: " + count);
                 SXRNode sceneObject = new SXRNode(sxrContext, sxrContext.createQuad(10f, 10f));
                 SXRRenderData renderData = sceneObject.getRenderData();
-                SXRTexture texture;
+                texture.clearData();
```
Comment (liaxim) [inline framework-tests/app/src/androidTest/java/com/samsungxr/stress/StressTests.java:151]: I find this clearData confusing. Immediately a new instance will be stored in "texture". If all is good, the old texture will be destroyed and dtor calls clearData. If there is really a need to call clearData explicitly here, it is a sign that something is not working as intended.

**Comment by liaxim** [comment]
I think the tests should not go into dedicated packages. They should share the main package as the framework to avoid exposing internal methods, like clearData, as public. There shouldn't be internal methods ideally but if we must, they better be hidden from the public API space.

**Pull request merged**